reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/client/JavaTCPClient.java

```diff
@@ -38,7 +38,7 @@ public void run(String realfile, String host, int port, boolean debug2, List<Str
             throws MalformedURLException, URISyntaxException {
         debug = debug2;
         // we pass in our reactive socket here to the test suite
-        String file = "reactivesocket-tck-drivers/src/test/resources/client$.txt";
+        String file = "reactivesocket-tck-drivers/src/test/resources/client.txt";
```

Comment:
This path looks wrong 

https://github.com/ReactiveSocket/reactivesocket-java/tree/master/reactivesocket-tck-drivers/src/test/resources